### PR TITLE
Add SRI to script tags where applicable

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description

Security wise all external scripts should have [SRI](https://infosec.mozilla.org/guidelines/web_security#subresource-integrity) enabled (aka subresource integrity). This allows the browser to check if the external source has been tampered with (and thus not load the script).

I've only added to those `<script>` sites that had the information publicly available and that pointed to a specific version of the resource.

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/views/articles/_hiring_form.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<script src="https://cdn.jsdelivr.net/npm/places.js@1.4.15"></script>
+<script src="https://cdn.jsdelivr.net/npm/places.js@1.4.15/dist/cdn/places.min.js"
```
Comment (Glennmen) [inline app/views/articles/_hiring_form.html.erb]: Wouldn't it be better to use CDNJS for all script tags? SRI is the same

```suggestion
<script src="https://cdnjs.cloudflare.com/ajax/libs/places.js/1.4.15/places.min.js"
```

**Comment by maestromac** [comment]
Hey @rhymes how did you generate those SHA-2s?

**Comment by rhymes** [comment]
@maestromac I didn't really, I went to https://cdnjs.com/ and copied them :P 

Screenshot 2019-03-22 at 7 21 49 PM

**Comment by rhymes** [comment]
You can also generate them [manually](https://infosec.mozilla.org/guidelines/web_security#examples-11) like this if you need to:

```shell
# Generate the hash myself
$ curl -s https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular.min.js | \
    openssl dgst -sha384 -binary | \
    openssl base64 -A

r1y8TJcloKTvouxnYsi4PJAx+nHNr90ibsEn3zznzDzWBN9X3o3kbHLSgcIPtzAp
```

**Comment by maestromac** [comment]
Oh, thanks for sharing that!

**Comment by rhymes** [comment]
@maestromac apparently there's also a web tool :D https://www.srihash.org/

**Pull request merged**